Server/service/pingService.js

```diff
@@ -19,9 +21,9 @@ const enqueueJob = async (queue, monitor) => {
       { monitor },
       { delay: monitor.interval * 1000 }
     );
-    logger.info("Job enqueued", { service: "pingService" });
+    logger.info("Job enqueued", { service: SERVICE_NAME });
```

Comment:
I suppose the reason for this is because a constant is better than a hard-coded string?

Reply:
Yep, don't want to update lots of hardcoded strings if we change the name of the service one day